### PR TITLE
Add conduit subscription support

### PR DESCRIPTION
**Pull request by FaeyUmbrea**
This PR should be all the required changes to allow creating EventSub subscriptions that use the new conduit transport.

### CODE REVIEW

**Review by swiftyspiffy — APPROVED** [review]
lgtm

File: TwitchLib.Api.Helix/EventSub.cs [hunk 1]
```diff
@@ -85,6 +86,22 @@ public EventSub(IApiSettings settings, IRateLimiter rateLimiter, IHttpCallHandle
                         }
                     };
                     return TwitchPostGenericAsync<CreateEventSubSubscriptionResponse>("/eventsub/subscriptions", ApiVersion.Helix, JsonConvert.SerializeObject(websocketBody), null, accessToken, clientId);
+                case EventSubTransportMethod.Conduit:
+                    if (string.IsNullOrWhiteSpace(conduitId))
+                        throw new BadParameterException("conduitId must be set");
+                    
+                    var websocketBody = new
```
Comment (Syzuna) [inline TwitchLib.Api.Helix/EventSub.cs:93]: the variable name has to be changed to sth else to avoid naming conflicts

**Review by Syzuna — CHANGES_REQUESTED** [review]

**Review by swiftyspiffy — CHANGES_REQUESTED** [review]
looks like build is failing, can you look into that

**Comment by FaeyUmbrea** [comment]
Seems to build now, at least locally

**Review by Syzuna — APPROVED** [review]
LGTM

**Review by swiftyspiffy — APPROVED** [review]

**Pull request merged**